fenix/app/src/main/java/org/mozilla/fenix/translations/TranslationOptionsDialog.kt

```diff
@@ -80,13 +119,41 @@ fun TranslationOptionsDialog(
 }
 
 @Composable
-private fun TranslationOptions(translationSwitchItem: TranslationSwitchItem) {
+private fun EnabledTranslationOptionsItems(
```

Comment:
Could you add some comments here too? Maybe a quick overview of how the logic works or behavior?

Reply:
Done